types/office-js-preview/index.d.ts

```diff
@@ -14493,6 +14502,54 @@ declare namespace Office {
          * @param userContext - Optional. Any state data that is passed to the asynchronous method.
          */
         getCallbackTokenAsync(callback: (asyncResult: Office.AsyncResult<string>) => void, userContext?: any): void;
+        /**
+         * Gets currently selected messages on which an add-in can activate and perform operations. An add-in can activate on a maximum of 100 messages at a time.
+         * To learn more about item multi-select, see
+         * [Activate your Outlook add-in on multiple messages (preview)](https://learn.microsoft.com/office/dev/add-ins/outlook/item-multi-select).
+         *
+         * @remarks
+         * [Api set: Mailbox preview]
+         *
+         * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: `ReadWriteMailbox`
+         *
+         * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose, Read
+         * 
+         * **Important**: This method only applies to messages.
+         *
+         * @param options - An object literal that contains one or more of the following properties.
+         *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
+         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter,
+         *        `asyncResult`, which is an `Office.AsyncResult` object. The properties of the selected messages are returned as an array of JSON objects in the
+         *        `asyncResult.value` property. These properties include the item ID, subject, item type (Message is the only supported type at this time), and item mode
+         *        (Read or Compose). The objects in the array follow the order in which messages were selected.
+         * 
+         * @beta
+         */
+        getSelectedItemsAsync(options: Office.AsyncContextOptions, callback: (asyncResult: Office.AsyncResult<object[]>) => void): void;
+        /**
+         * Gets currently selected messages on which an add-in can activate and perform operations. An add-in can activate on a maximum of 100 messages at a time.
```

Comment:
It's not clear to me what "activate" means here. Does this feature require the use of activation rules, such as described in https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/activation-rules ?

Reply:
In this context, "activate" refers to the add-in "lighting up"/becoming active in the ribbon when multiple messages are selected (as opposed to it appearing grayed out). Multi-select doesn't require the use of activation rules.

Reply:
got it.